plugins/menus/src/ImportSessionWidget/components/ImportSessionWidget.js

```diff
@@ -109,17 +125,19 @@ function ImportSession(props) {
       let sessionContents
       try {
         sessionContents = JSON.parse(sessionText).session
+        setSessionToActivate(JSON.stringify(sessionContents))
+        setDisplayConfirm(true)
       } catch (error) {
         console.error(error)
         sessionContents = { error: `Error parsing ${file.path}: ${error}` }
       }
-      const session = getSession(model)
-      try {
-        session.setSession(sessionContents)
-      } catch (error) {
-        console.error(error)
-        setErrorMessage(`Error activating session: ${error} `)
-      }
+      // const session = getSession(model)
+      // try {
+      //   session.setSession(sessionContents)
+      // } catch (error) {
+      //   console.error(error)
+      //   setErrorMessage(`Error activating session: ${error} `)
+      // }
```

Comment:
Why is this part removed/commented out?

Reply:
It's commented out because that causes the automatic import. Now it waits for the user feedback to click confirm before running that code so it's on the onclick now